script.js

```diff
@@ -1,4 +1,109 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  console.log(`Round is ${roundCounter}`);
```

Comment:
good use of console.log statements for debugging but they are usually removed before shipping to production in a professional setting (the equivalent of submitting your project for this course)